elastic/kibana
# [Timelion] Add telemetry for scripted fields

Closes: #91376

## Summary

Added telemetry for scripted fields. Only visualizations that have changed in the last 90 days are taken into account (the way to count only active visualizations. We can discuss it and choose another period).

### How to test it:

- create a timelion visualization with scripted field and save it
- open `/api/stats?extended=true` in browser
- find `vis_type_timelion` property
- you can see the following in stats:
![image](https://user-images.githubusercontent.com/16915480/109946264-1a930900-7ce9-11eb-8059-89fabf90cfb7.png)


### Checklist

Delete any items that are not applicable to this PR.
